src/client/cassh

```diff
@@ -80,6 +80,13 @@ def read_conf(conf_path):
         user_metadata['auth'] = None
         user_metadata['realname'] = None
 
+    try:
+        user_metadata['ldap_enable'] = bool(config.get('ldap', 'enable') != 'False')
```

Comment:
Couldn't we just check if there is a section 'ldap' in the config file rather than adding a variable / param in the config ?